src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,19 +1,48 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    public static final Integer STORE_SIZE = 10;
+    private Object[] keyArr;
+    private Object[] valueArr;
+    private int lengthOfActualValue = 0;
+
+    public StorageImpl() {
+        valueArr = (V[]) new Object[STORE_SIZE];
+        keyArr = (K[]) new Object[STORE_SIZE];
+    }
+
     @Override
     public void put(K key, V value) {
+        for (int i = 0; i < lengthOfActualValue; i++) {
+            if (Objects.equals(keyArr[i], key)) {
+                valueArr[i] = value;
+                return;
+            }
+        }
+        keyArr[lengthOfActualValue] = key;
+        valueArr[lengthOfActualValue] = value;
+        lengthOfActualValue++;
     }
 
     @Override
     public V get(K key) {
+        int index = -1;
+        for (int i = 0; i < lengthOfActualValue; i++) {
+            if (Objects.equals(keyArr[i], key)) {
+                index = i;
+            }
+            if (index >= 0) {
+                return (V) valueArr[index];
```

Comment:
You can make code more readable and efficient
```suggestion
                if (Object.equals(keyArr[i], key)) {
                return (V)` valueArr[i];`
                }
```